include/common.sh

```diff
@@ -211,14 +211,31 @@ deref()
   fi
 }
 
+#return 1 if brightness is dark, 0 if light, and -1 if unknown (considered to be dark as default)
+darkbackground() {
+  if [ "${#COLORFGBG}" -ge 3 ]; then
+    bg=${COLORFGBG##*;}
+    if [ ${bg} -lt 7 ]; then
+      return 1
+    else
+      return 0
+    fi
+  else
+    return -1
+  fi
+}
+
 defineANSI()
 {
   # Standard tty codes
   ESC="\047"
   ERASELINE="${ESC}[2K"
   CRSRHIDE="${ESC}[?25l"
   CRSRSHOW="${ESC}[?25h"
-
+  
```

Comment:
I'd recommend deleting these newlines

Reply:
659e5746f44520595d5c06bf011200e2409144f0
Removed the newlines